_posts/2018-12-10-blog-lets-create some-glip-bots-with-python.markdown

```diff
@@ -0,0 +1,321 @@
+---
+layout: post
+title:  "Let's create some glip bots with python"
+date:   2018-12-10 13:31:07 +0800
+categories: blog
+---
+
+![ ](https://github.com/zxdong262/ringcentral-chatbot-factory-py/blob/master/screenshots/wanted.jpg?raw=true)
+
+This is a intro blog about my recent work: [ringcentral-chatbot-python](https://github.com/zxdong262/ringcentral-chatbot-python).
+
+The project is part of our chatbot plan: Make writing a glip chatbot easy and fun. It is not mature enough for public yet, still need more documents and adding more features.
```

Comment:
Don't have to capitalize after colon (':').
When referring to an thing to do, either use 'to make writing ....' or 'making writing ...'.
I am not sure how to explain this, but 'It is not mature enough for public yet, needing more documents and more features added still.' would sound more appropriate.